src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import commentsFromServer from '../../api/comments';
+
+import { Post } from '../../Types/post';
+import { Comment } from '../../Types/comment';
+
+import { PostInfo } from '../PostInfo';
+
+function getCommit(postId: number): Comment[] {
```

Comment:
The function name 'getCommit' is misleading, consider renaming it to 'getCommentsByPostId' for better clarity.